schemas/v3.1/meta/base.schema.json

```diff
@@ -0,0 +1,78 @@
+{
+    "$id": "https://spec.openapis.org/oas/3.1/meta/base",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$vocabulary": {
+        "https://spec.openapis.org/oas/3.1/vocab/base": true
+    },
+    "$dynamicAnchor": "meta",
+    "title": "OAS Base vocabulary",
+
+    "type": ["object", "boolean"],
+    "properties": {
+        "example": true,
+        "discriminator": { "$ref": "#/$defs/Discriminator" },
+        "externalDocs": { "$ref": "#/$defs/ExternalDocs" },
+        "xml": { "$ref": "#/$defs/Xml" }
+    },
+    "$defs": {
+        "Extensible": {
+            "patternProperties": {
+                "^x-": true
+            }
+        },
+        "Discriminator": {
+            "allOf": [{ "$ref": "#/$defs/Extensible" }],
```

Comment:
Since `$ref` can appear alongside other keywords the `allOf` isn't necessary. It's there a reason to keep it?

Reply:
Good point. That was force of habit :smiley: